src/conditions/base/rpc.ts

```diff
@@ -18,15 +18,17 @@ const makeParameters = () =>
     })),
   });
 
+export const rpcConditionSchema = {
+  chain: Joi.number()
+    .valid(...SUPPORTED_CHAINS)
+    .required(),
+  method: Joi.string()
+    .valid(...Object.keys(rpcMethodSchemas))
+    .required(),
+  parameters: makeParameters(),
```

Comment:
Is this `makeParameters()` function going away in a different PR? We chatted about other rpc methods being allowed...

I assume with it going away it woudn't have to be redefined for `EvmCondition`. Also, side note: `EvmCondition` got changed to `ContractCondition` on the python-side. I think just to open up the name a bit more.